sdk/spring-sdk/src/it/java/com/example/wiring/workflowentities/SpringWorkflowIntegrationTest.java

```diff
@@ -26,6 +26,7 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.annotation.Import;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
```

Comment:
why that changed? 

probably `org.springframework.http.HttpStatus` is not in use (line above).

Reply:
Is that because of Spring 3.0.2?

Reply:
yes